containers/python/python_container.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import print_function
+import rpc
+import os
+import sys
+
+import numpy as np
+np.set_printoptions(threshold=np.nan)
+
+sys.path.append(os.path.abspath("/lib/"))
+import pywrencloudpickle
+
+
+def load_predict_func(file_path):
+    with open(file_path, 'r') as serialized_func_file:
+        return pywrencloudpickle.load(serialized_func_file)
+
+
+class PredictContainer(rpc.ModelContainerBase):
+    def __init__(self, path, input_type):
+        print("Initializing prediction container")
+
+        self.input_type = rpc.string_to_input_type(input_type)
+
+        predict_fname = "predict.txt"
+        predict_path = "{dir}/{predict_fname}".format(
+            dir=path, predict_fname=predict_fname)
+
+        self.predict_func = load_predict_func(predict_path)
+
+        print("Loaded prediction function")
+
+    def predict_ints(self, inputs):
+        if self.input_type != rpc.INPUT_TYPE_INTS:
+            pass
```

Comment:
you should probably print an error in the case of a type mismatch instead of silently passing